src/test/java/io/pravega/connectors/hadoop/PravegaConnectorLocalJobITCase.java

```diff
@@ -259,22 +265,25 @@ public void testPravegaConnectorOutput() throws Exception {
         }
     }
 
-    private Job configureJobWithInputAndOutput(Configuration conf, Path outputPath) throws Exception {
+    private Job configureJobWithInputAndOutput(Configuration conf, Path outputPath, String startPos, String endPos) throws Exception {
         conf = PravegaInputFormat.builder(conf)
```

Comment:
The code below gives an impression that conf is being replaced in line with another instance, or maybe it is progressively building it. The two blocks seem to be redundant, or at least confusing. 

Also, the method name says configureJob..., but it is also preparing a configuration: I'd suggest breaking it down into two separate methods - one that builds a configuration object, and a second that configures the Job with Input and Output with the provided configuration object. If you do that, adding security configuration to the conf object should also become the responsibility of the former method.

Reply:
reorganized the code to draw the configurations using separate methods.